src/data/events.json

```diff
@@ -174,5 +174,17 @@
     "location": "Chennai",
     "communityName": "TechNexus Community",
     "communityLogo": "https://secure.meetupstatic.com/photos/event/5/2/7/5/clean_525381109.webp"
-  }
+  },
+  {
+  "eventName": "Swift Chennai Chapter #01",
+  "eventDescription": "Swift Chennai ​Chapter #01 marks the start of a fresh, vibrant meetup series for Swift devs, iOS enthusiasts, and curious creators. Come for the code, stay for the kaapi, and leave with solid connections.",
+  "eventDate": "2025-04-26", // Format: YYYY-MM-DD
```

Comment:
So we don't usually use // comments in our json file and that kind of broke the page : )